src/main/java/seedu/address/model/relationship/RelationshipDirection.java

```diff
@@ -0,0 +1,29 @@
+package seedu.address.model.relationship;
+
+/**
+ * This class defines the direction of relationships for Relationship class
+ */
+public enum RelationshipDirection {
+
+    UNDIRECTED("undirected"),
+    DIRECTED("directed");
```

Comment:
This is not how to use enums. These properties can be referred to simply as `RelationshipDirection.UNDIRECTED` and `RelationshipDirection.DIRECTED`, and to check direction, you can simply compare with these values.